docs/releases/1.26-NOTES.md

```diff
@@ -8,10 +8,10 @@ This is a document to gather the release notes prior to the release.
 
 * The default networking provider for new clusters is now Cilium.
 
-* Control-plane instance groups in new clusters are now created with names starting
+* Control plane instance groups in new clusters are now created with names starting
```

Comment:
```suggestion
* Control-plane instance groups in new clusters are now created with names starting
```

Hyphens should be used with compound modifiers.

Reply:
I was looking at https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication, where they spell it "control plane".

Reply:
That document doesn't use the term as a compound modifier. When it is used as a compound modifier, English grammar requires it be hyphenated.